lib/task.js

```diff
@@ -22,6 +27,23 @@ export class TaskClient extends Client {
             .then(res => res.data)
             .then(data => this.afterDataProcessed(endpoint, data));
     }
+
+    /**
+     * NOT for use in kit app ui
+     * Get delete account auth via token is currently only used on the guardian website
+     * @param token One use token provided by the consent email
+     * @returns token to be used for account deletion
+     */
+    getDeleteAccountAuthViaToken(token) {
+        const endpoint = this.getEndpoint('getDeleteAccountAuthViaToken');
+        if (token) {
+            endpoint.headers.append('Authorization', `Bearer ${token}`);
+        }
+
+        return this.fetch(endpoint)
+            .then(res => res.data)
```

Comment:
is this line needed or can we just do:
`.then(res => doSomethingWith(res.data))`?

Reply:
it just follows the way the rest of the endpoints have been implemented